### DISCUSSION TITLE
Error every time i try to open colmap

### DISCUSSION THREAD

**Discussion by AnvithPabba** [colmap/colmap#2498] [General]
Hi, every time I try to double click COLMAP.bat, I get an error saying "The application was unable to start correctly (0x0000279). Click OK to close the application."

I am on windows 11, surface laptop studio, and I had to manually unblock this from windows defender to get it running. Can anyone tell me what I'm doing wrong and how do I fix this?

Thank you.

<img width="861" alt="error_colmap" src="https://github.com/colmap/colmap/assets/87067337/8c795a0c-4b3c-4859-b0b9-1b19cc93c96f">

**Comment by Kazuhira-Okada** [comment]
Can you check any error logs relating colmap in windows event viewer?
